mail_restrict_follower_selection/__init__.py

```diff
@@ -0,0 +1,20 @@
+##############################################################################
```

Comment:
Should we switch to the shorter version?

```
# 2015 Therp BV <http://therp.nl>
# 2017 Komit <http://www.komit-consulting.com>
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).`
```

Also, I am not sure if it is ok to add Komit  to the copyrights for a migration.

@OCA/core-maintainers should there be any mention about the good practice in the migration guideline page: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0 ?

Reply:
In pull request https://github.com/OCA/web/pull/738, @moylop260 asked me to keep original Copyright so i don't want to switch this to the shorter version.

Reply:
it's fine to shorten this, `keeping the copyright` means not to delete the names. So I agree with @jcdrubay's proposal